docs/migration-guide/to-16.md

```diff
@@ -1,33 +1,58 @@
 
 ## Removed deprecated stylistic rules
 
-We've removed the stylistic rules we deprecated in 15.0.0. See the [15.0.0 migration guide](./to-15.md) for details on how to migrate.
+We've removed the stylistic rules we deprecated in 15.0.0.
+
+You should remove the rules from your configuration object. See the [15.0.0 migration guide](./to-15.md#deprecated-stylistic-rules) for more details.
 
 ## Removed support for Node.js 14
 
-Node.js 14 has reached end-of-life. We've removed support for it so that we could update some of our dependencies. You should use the following or higher versions of Node.js:
+Node.js 14 has reached end-of-life. We've removed support for it so that we could update some of our dependencies.
+
+You should use the following or higher versions of Node.js:
 
 - 16.13.0
 - 18.0.0
 
-## Changed CLI exit code for flag error
-
-We've changed the exit code for CLI flag errors from `2` to `64` so that `2` is only used for lint problems.
-
-## Changed CLI to print problems to stderr instead of stdout
-
-If you use the CLI to fix a source string by using the [`--fix`](../user-guide/cli.md#--fix) and [`--stdin`](../user-guide/cli.md#--stdin) options, the CLI will print the fixed code to stdout and any problems to stderr.
-
 ## Changed Node.js API returned resolved object
 
 We changed the resolved object of the Promise returned by `stylelint.lint()` so that:
 
 - the [`output`](../user-guide/node-api.md#output) property now only contains formatted problems
 - a new [`code`](../user-guide/node-api.md#code-1) property contains the autofixed code
 
-This means if you use `stylelint.lint()` to lint a source string and the `fix` option is set to `true`, the `output` property will be the formatted problems and the `code` property will be the fixed code.
+If you use `stylelint.lint()` to lint a source string and the `fix` option is `true`, the `output` property will contain the formatted problems and the `code` property will contain the fixed code.
+
+```diff js
+async function lint() {
+  const result = await stylelint.lint({
+    code: "a {}",
+    fix: true
+  });
+- const fixedCode = result.output;
++ const fixedCode = result.code;
+}
+```
+
+If you use `stylelint.lint()` to lint files, the `code` property will always be `undefined`.
+
+## Changed CLI to print problems to stderr
+
+We've changed the CLI to print problems to stderr instead of stdout.
+
+If you use the [`--fix`](../user-guide/cli.md#--fix) and [`--stdin`](../user-guide/cli.md#--stdin) options, the CLI will print the fixed code to stdout and any problems to stderr.
+
+## Changed CLI exit code for flag errors
+
+We've changed the exit code for CLI flag errors from `2` to `64` so that `2` is only used for lint problems.
 
-If you use `stylelint.lint()` to lint files, then the `code` property will always be `undefined`.
+If you're an author of an editor integration that uses the CLI, you can now distinguish between flag errors and lint problems.
```

Comment:
Maybe we need to sync the releases of certain extensions with v16?
e.g. coordinate with @ota-meshi for stylelint/vscode-stylelint

Reply:
In fact, it seems difficult since we have not maintained `vscode-stylelint` well recently. 😓

Reply:
Yeah, I think we need to change vscode-stylelint.